scipy/linalg/decomp_svd.py

```diff
@@ -279,6 +280,97 @@ def diagsvd(s, M, N):
         raise ValueError("Length of s must be M or N.")
 
 
+def hosvd(a, full_tensor=True, check_finite=True):
+    """
+    Higher-order SVD (HOSVD)
+
+    Factorizes the M-D tensor `a` into a list of M matrices ``U_k`` containing
+    the left singular vectors of the unfolded tensor with respect to `k`\\ th
```

Comment:
Looks like a typo? Not sure how this should render for you. But as it's not a parameter of the function it should have a double tick.

Reply:
The highlighting of the `k` was by purpose. I fixed that double ticks are used.